composer/core/state.py

```diff
@@ -282,6 +287,7 @@ def __init__(
             "timestamp",
             "rank_zero_seed",
             "current_metrics",
+            "run_name",
```

Comment:
Can you also add the `run_name` into the table of attributes that get serialized?